deploy/charts/impt/chart/Chart.yaml.template

```diff
@@ -0,0 +1,138 @@
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/resource/.build/chart"
+    condition: enabled.resource
+  - name: director
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/director/.build/chart"
+    condition: enabled.director
+  - name: dataset-ie
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/dataset_ie/.build/chart"
+    condition: enabled.dataset-ie
+  - name: media-ms
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/media/.build/chart"
+    condition: enabled.media-ms
+  - name: auto-train-controller
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/auto_train/.build/chart"
+    condition: enabled.auto-train-controller
+  - name: inference-gateway
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/inference_gateway/.build/chart"
+    condition: enabled.inference-gateway
+  - name: jobs-execution-namespace
+    version: v0.1.15
+    condition: enabled.jobs-execution-namespace
+  - name: jobs
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/jobs/.build/chart"
+    condition: enabled.jobs
+  - name: project-ie
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/project_ie/.build/chart"
+    condition: enabled.project-ie
+  - name: visual-prompt
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/visual_prompt/.build/chart"
+    condition: enabled.visual-prompt
+  - name: platform-cleaner
+    version: ${TAG}
+    repository: "file://../../../../../platform/services/platform_cleaner/.build/chart"
+    condition: enabled.platform-cleaner
+  - name: telemetry-compression
+    version: 0.0.1
+    condition: enabled.telemetry-compression
+  - name: spice-db
+    version: 0.0.5
+    condition: enabled.spice-db
+  - name: postgresql
+    version: 0.0.3
+    condition: enabled.postgresql
+  - name: seaweed-fs
+    version: 0.0.11
+    condition: enabled.seaweed-fs
+  - name: opentelemetry-collector
+    version: 0.0.4
+    condition: enabled.opentelemetry-collector
+  - name: secrets
+    version: 0.0.5
+    condition: enabled.secrets
+  - name: dcgm-exporter
+    version: 3.4.1
+    condition: enabled.dcgm-exporter
+  - name: xpu-manager
+    version: 0.0.1
+    condition: enabled.xpu-manager
+  - name: etcd
+    version: 9.5.0
+    condition: enabled.etcd
+  - name: cert-manager
+    version: 0.1.1
+    condition: enabled.cert-manager
+  - name: modelmesh-serving
+    version: 0.1.0
+    condition: enabled.modelmesh-serving
+  - name: modelregistration
+    version: ${TAG}
+    repository: "file://../../../../../interactive_ai/services/model_registration/.build/chart"
+    condition: enabled.modelregistration
+  - name: cert-manager-istio-csr
+    version: v0.13.0
+    condition: enabled.istio-csr
+  - name: opa-bundle
```

Comment:
if we are using a hardcoded opa-bundle chart, let's remove this one https://github.com/open-edge-platform/geti/tree/jchrapko/geti_installer/platform/services/opa_bundle

But please make sure it's ok to loose these changes that Lena made recently to it
https://github.com/open-edge-platform/geti/pull/861